docs/MO_DG/prepare_model/convert_model/Converting_Model_General.md

```diff
@@ -59,11 +59,14 @@ Framework-agnostic parameters:
   --reverse_input_channels
                         Switch the input channels order from RGB to BGR (or
                         vice versa). Applied to original inputs of the model
```

Comment:
@nosovmik, why do we decide to stay "Applied to original inputs of the model..." instead of "Applied to user's input data..."?

Reply:
It is a legacy behavior, and it is consistent with model's documentation. E.g. if model is RGB and states that means (120, 130, 140) shall be applied, but user has BGR input - in this case user will need to `--mean_values(120,130,140) --reverse_input_channels` instead of converting something in the mind and have `--mean_values(140,130,120) --reverse_input_channels`

Reply:
Let us merge it and continue this discussion offline